SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/GeneratedTestMap.cpp

```diff
@@ -24,26 +24,37 @@ UGeneratedTestMap::UGeneratedTestMap()
 	BasicShapeMaterial = MaterialAsset.Object;
 }
 
-UGeneratedTestMap::UGeneratedTestMap(EMapCategory InMapCategory, FString InMapName)
+UGeneratedTestMap::UGeneratedTestMap(const EMapCategory InMapCategory, const FString& InMapName)
 	: UGeneratedTestMap()
 {
 	MapCategory = InMapCategory;
 	MapName = InMapName;
 	bIsValidForGeneration = true;
 }
 
+void UGeneratedTestMap::Init(const EMapCategory InMapCategory, const FString& InMapName)
```

Comment:
Are you sure you need this? Can't all use cases just call the constructor above?

Reply:
related to above comment on idk how to